be/src/common/config.h

```diff
@@ -818,4 +818,11 @@ CONF_Int32(cardinality_of_inject, "100");
  * Split by "," strictly.
  */
 CONF_String(directory_of_inject, "/src/exec,/src/exprs");
+
+// Used by to_base64
+CONF_Int64(max_length_for_to_base64, "200000");
+// Used by bitmap_to_string
+CONF_Int64(max_length_for_bitmap_to_string, "200000");
```

Comment:
bitmap_to_string and bitmap_to_array only need one config.  could be 1000000.

Reply:
done.